addons/mail/static/src/models/activity_view/activity_view.js

```diff
@@ -66,6 +66,17 @@ registerModel({
             }
             return _.str.sprintf(this.env._t("for %s"), this.activity.assignee.nameOrDisplayName);
         },
+        /**
+         * @private
+         * @returns {ClockWatcher}
+         */
+        _computeClockWatcher() {
+            return insertAndReplace({
+                clock: insertAndReplace({
+                    frequency: 60 * 1000,
+                }),
+            });
+        },
```

Comment:
Minor details:

The method returns a command not a ClockWatcher.

Also this could probably be a default instead as all values are static.